node-runner-cli/utils/Prompts.py

```diff
@@ -0,0 +1,300 @@
+import os
+
+from env_vars import SUPPRESS_API_COMMAND_WARN
+from utils.utils import Helpers, run_shell_command, bcolors
+
+
+class Prompts:
```

Comment:
From a clean code perspective, it feels weird to me that these methods which are used in 1 place are all located in a separate (Prompts) class

Reply:
As these are user facing instructions, I thought i better pull them out into a separate file. I felt at times if these questions stay along with code, it wasn't that readable

Reply:
Makes sense :)